packages/Graph/Actions/User/GetDirectReports.cs

```diff
@@ -51,7 +51,11 @@ internal override async Task<IEnumerable<DirectoryObject>> CallGraphServiceWithR
             int maxCount = (int)parameters["MaxResults"];
 
             // Create the request first then apply the Top() after
-            IUserDirectReportsCollectionWithReferencesRequest request = client.Users[userId].DirectReports.Request();
+            // TODO: Make this SELECT() clause configurable for different column names
+            // EXPLAINER: The reason we are limiting the field is for two reasons:
+            //            1. Limit the size of the graph object payload needed to be transferred over the wire
+            //            2. Reduces the exposure of end user PII (Personally Identifiable Information) in our system for privacy reasons. It's generally good practice to use what you need.
+            IUserDirectReportsCollectionWithReferencesRequest request = client.Users[userId].DirectReports.Request().Select("id,displayName,mail,businessPhones,department,jobTitle,officeLocation");
```

Comment:
I don't think you should forcibly limit what fields are selected at this layer. This component should be designed for generic usage, and there may be valid reasons to require other fields. I think it's fine to have sensible defaults, but I would also ensure that it can overwritten.

I also wouldn't include the second comment - again, this is a generic component that others may use for their own purposes. There's no harm in interacting with PII as long as we understand what we are doing with it. We shouldn't lump in our own concerns with things like the Calendar, People and Enterprise Assistant template into this.

Reply:
Makes sense. I just wanted to get unblocked on this first and then work on a second PR that includes a regenerated schema, etc. I guess I can make that change now.